src/CommunityToolkit.Maui/Controls/Avatar.cs

```diff
@@ -0,0 +1,557 @@
+using System.ComponentModel;
+using Microsoft.Maui.Controls.Internals;
+using Microsoft.Maui.Controls.Shapes;
+
+namespace CommunityToolkit.Maui.Controls;
+
+/// <summary>Avatar content view.</summary>
+public class Avatar : ContentView, IAvatarElement, IFontElement, ITextElement, IImageElement
+{
+	/// <summary>The backing store for the <see cref="IAvatarElement.AvatarBackgroundColor" /> bindable property.</summary>
+	public static readonly BindableProperty AvatarBackgroundColorProperty = BindableProperty.Create(nameof(IAvatarElement.AvatarBackgroundColor), typeof(Color), typeof(IAvatarElement), null, propertyChanged: OnBackgroundColorChanged);
+
+	/// <summary>The backing store for the <see cref="IAvatarElement.AvatarHeightRequest" /> bindable property.</summary>
+	public static readonly BindableProperty AvatarHeightRequestProperty = BindableProperty.Create(nameof(IAvatarElement.AvatarHeightRequest), typeof(double), typeof(VisualElement), defaultValue: AvatarElement.DefaultHeightRequest, propertyChanged: OnHeightRequestPropertyChanged);
+
+	/// <inheritdoc/>
+	public static readonly BindableProperty AvatarPaddingBottomProperty = BindableProperty.Create("AvatarPaddingBottom", typeof(double), typeof(IPaddingElement), default(double), propertyChanged: OnPaddingBottomChanged);
+
+	/// <inheritdoc/>
```

Comment:
What is this XML inheriting from?

Reply:
My apologies. This was developer laziness and I failed to go back and implement correctly. Will implement correctly.